src/Search/FlagrowIOSearcher.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Flagrow\Bazaar\Search;
+
+use Flagrow\Bazaar\Extensions\Extension;
+use Flarum\Core\Search\SearchResults;
+use Flarum\Extension\ExtensionManager;
+use Flarum\Settings\SettingsRepositoryInterface;
+use GuzzleHttp\Client;
+use Illuminate\Database\Eloquent\Collection;
+
+class FlagrowIOSearcher extends AbstractExtensionSearcher
```

Comment:
I tried something by doing an abstract `Searcher` and then one for our specific endpoint. That's why the name is in it. (It's not only Flagrow, it's `FlagrowIO`)

Of course if we only ever use one source of data, I could remove the `abstract` and put that code in it.

Don't know if we plan to allow extensibility on this. At the moment it's possible to register another `Seacher` in place of this one through the IOC.